### ISSUE TITLE
Switching tabs after unlocking database

### ISSUE THREAD

**Issue by Groundrunner** [subdavis/Tusk#95]
## This issue is a 

- [x] bug
- [ ] feature
- [ ] question

### Please describe the current behavior, and explain why it's bad.

After unlocking my database using my master password, if I then switch to a different tab and click the extension icon, the Tusk extension is stuck in the "unlocking" state (rotating blue circle). If I return to the first tab and click the icon, the extension also shows the "unlocking" state. It remains in this state for about 1 minute. After that I am again able to click the icon and enter my master password.
This basically means, that if I for some reason or another leave the extension dialog after unlocking the database, I will have to wait 1 minute to open it again.

So short version for reproducability:

1. Click icon to open extension dialog.
2. Enter master password
3. Open new tab in browser.
4. Return to previous tab and click icon to open extension dialog.
5. The dialog is now stuck in "unlocking stage".

Browser: Chrome Version 63.0.3239.132 (Officiel version) (64-bit)
Tusk version: 2018.1.31
Database location: Dropbox 

Thank you for your work! I wouldn't be able to use keepass without it.

**Comment by subdavis** [comment]
Thanks for the detailed report.  I'll take a look first thing when I get home tonight.  Ty!

**Comment by subdavis** [comment]
This is a tough one, @Groundrunner!

I've traced the error to an uncaught exception in `window.crypto.subtle.decrypt` in the background page.  This is throwing a generic `DOMException`.  I'll need to test with a previous version of Chrome, because I believe this may actually be a chrome bug.

I'm also on version ` 63.0.3239.132` 64 bit for Linux.  Are you by chance using the Linux variant?

**Comment by subdavis** [comment]
Good news.  There's some randomness involved here too.  1 in about 20 attempts actually succeeds, so I'm guessing it has to do with key generation.

Created a JS fiddle @ https://jsfiddle.net/8wbLyLfc/30/ to try and track my progress on a proof of concept  bug, but have yet to find it.

**Comment by Groundrunner** [comment]
I am happy that you are looking into it. Just for info, I am using a 64 bit version of Windows 10.

Good luck.

**Comment by subdavis** [comment]
Solved in #96 
Caused by race condition in key reset.
Should be pushed to the webstore later this weekend.  Thanks again.
